autrainer-configurations/dataset/MSPPodcast-EmoClass-big4-wav.yaml

```diff
@@ -0,0 +1,30 @@
+# Important: should be used with inference_batch_size: 1
+id: MSPPodcast-EmoClass-big4-wav
+_target_: autrainer.datasets.MSPPodcast
+
+path: data/MSPPodcast
+features_subdir: Audios
+index_column: FileName
+target_column: EmoClass
+file_type: wav
+file_handler: autrainer.datasets.utils.AudioFileHandler
+
+criterion: autrainer.criterions.BalancedCrossEntropyLoss
+metrics: 
+  - autrainer.metrics.Accuracy
+  - autrainer.metrics.UAR
+  - autrainer.metrics.F1
+tracking_metric: autrainer.metrics.Accuracy
+
+categories: A,H,N,S
```

Comment:
The categories would be passed as the string `"A,H,N,S"` here, so this needs to be changed to `categories: [A,H,"N",S]`.